plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -315,15 +315,15 @@ func ClusterRoles() []rbacv1.ClusterRole {
 					"deployments", "deployments/scale",
```

Comment:
`statefulsets/status`?

Reply:
add "daemonsets/status", "deployments/status", "replicasets/status" here for the apps API group as well